titles/getting-started-rhdh/title-getting-started.adoc

```diff
@@ -16,6 +16,9 @@ include::modules/installation/con-rhdh-overview.adoc[leveloffset=+1]
 // installation (links to admin guide)
 include::modules/installation/proc-install-rhdh.adoc[leveloffset=+1]
 
+// Sizing
+include::installation/ref-rhdh-sizing.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/installation/ref-rhdh-sizing.adoc[leveloffset=+1]
```

Looks like the file path cannot be resolved in the preview website: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-221/getting-started-rhdh/#proc-install-rhdh_rhdh-getting-started

```
Unresolved directive in title-getting-started.adoc - include::installation/ref-rhdh-sizing.adoc[leveloffset=+1]
```

Reply:
Thanks @rm3l. I've fixed it.